crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/service/MaatCourtDataServiceIntegrationTest.java

```diff
@@ -128,7 +135,11 @@ public void whenGetPassportAssessmentFromRepIdAPICalled_thenTheCorrectResponseIs
     @Test
     public void whenGetPassportAssessmentFromRepIdAPICalled_thenInvalidResponseErrorsAreHandledCorrectly() {
         setupMockInvalidResponse();
-        APIClientException error = assertThrows(APIClientException.class, () -> maatCourtDataService.getPassportAssessmentFromRepId(repId, laaTransactionId));
+        APIClientException error = assertThrows(
+                APIClientException.class,
+                () ->
+                        maatCourtDataService.getPassportAssessmentFromRepId(repId, laaTransactionId)
```

Comment:
line breaks not required as it does not exceed the column width of 120 characters